src/collectd.conf.pod

```diff
@@ -7136,15 +7136,15 @@ read function is blocking, you should keep this value as low as possible. Keep
 in mind that the sum of all B<Timeout> values for all B<Nodes> should be lower
 than B<Interval> defined globally.
 
-=item B<Db> I<Db>
-
-Database to perform queries
-
 =item B<Query> I<Querystring>
 
 The B<Query> block identifies a query to execute against the redis server.
 There may be an arbitrary number of queries to execute.
 
+=item B<Db> I<Db>
+
+Database to perform query
```

Comment:
Too short and is out of style.

Reply:
Allows to select the Redis logical database by its zero-based numeric index I<Database>.
Database with index 0 is used by default.